packages/light-client/src/utils/utils.ts

```diff
@@ -78,3 +80,19 @@ export function isEmptyHeader(header: BeaconBlockHeader): boolean {
 // Thanks https://github.com/iliakan/detect-node/blob/master/index.esm.js
 export const isNode =
   Object.prototype.toString.call(typeof process !== "undefined" ? process : 0) === "[object process]";
+
+export async function getGenesisData(api: Pick<Api, "beacon">): Promise<GenesisData> {
+  const res = await api.beacon.getGenesis();
+  ApiError.assert(res);
+
+  return {
+    genesisTime: res.response.data.genesisTime,
+    genesisValidatorsRoot: res.response.data.genesisValidatorsRoot,
+  };
+}
+
+export async function getFinalizedSyncCheckpoint(api: Pick<Api, "beacon">): Promise<Bytes32> {
+  const res = await api.beacon.getStateFinalityCheckpoints("head");
```

Comment:
Should we add a check here to make sure it returns the finalized head. 

As per the spec the endpoint can return zero hash if finality not achieved.

Reply:
I would then expect that the `root` is an empty `Bytes32` (i.e. `Uint8Array`)?
Alternatively, we can return null in this case.

Reply:
That's totally an edge case for testing, when you have a custom new network. So I would skip this point for now.